### ISSUE TITLE
Error when compiling embedded extractor on coral

### ISSUE THREAD

**Issue by LukasLoacker** [f0cal/google-coral#17]
I am trying to retrain an image classifier on my coral edgeTPU device. Therefore I followed the steps explained on the <a href="https://coral.withgoogle.com/docs/edgetpu/retrain-classification-ondevice/">Coral's "Retrain an image classification model on-device"  tutorial</a>: 

## embedded extractor creation ##

I used the <a href="http://download.tensorflow.org/models/mobilenet_v1_2018_08_02/mobilenet_v1_1.0_224_quant.tgz">quantized model</a> from <a href="https://www.tensorflow.org/lite/guide/hosted_models">tensorflow</a>.

As a matter of fact, I created an embedding extractor tflite file according to the given example: 

```
tflite_convert \
--output_file=mobilenet_v1_embedding_extractor.tflite \
--graph_def_file=mobilenet_v1_1.0_224_quant_frozen.pb \
--input_arrays=input \
--output_arrays=MobilenetV1/Logits/AvgPool_1a/AvgPool
```

## Edge TPU Model Compiler upload ##
I got the file ```mobilenet_v1_embedding_extractor.tflite``` and uploaded it into the ```Edge TPU Model Compiler``` . Unfortunately the compiling process does fail and I get the following error message: 

```

ERROR: Something went wrong. Couldn't compile model.

More details
--------------
Start Time     2019-05-02T14:14:53.309219Z
State          FAILED
Duration       5.963912978s
Type           type.googleapis.com/google.cloud.iot.edgeml.v1beta1.CompileOperationMetadata
Name           operations/compile/16259636989695619987

```

(The compling works without any errors, when i use the unmodified ```mobilenet_v1_1.0_224_quant.tflite``` model.)

From my point of understanding the above mentioned procedure has to be accomplished before the on-device learning with the ```classification_transfer_learning.py``` script is executed on the raspberryPi + edgeTPU/ devBoard.

*I hope you can give me a hint to solve the problem and thanks in advance.*

**Comment by Namburger** [comment]
@LukasLoacker humm, I think this is the wrong usage for the embedding extractor...
The tutorial states that these model were created in order for us to append the softmax layer to it. It is not meant to be compiled or use before appending that layer. If you follow the tutorial, the final model that you've finished retraining should be called `retrained_model.tflite`
